### ISSUE TITLE
Pasting in reverse with right-click

### ISSUE THREAD

**Issue by tylorhl** [PowerShell/PowerShell#3816]
Steps to reproduce
------------------
Copy text with LF line endings and paste into console with right-click

Expected behavior
-----------------
Text is pasted in its original order

Actual behavior
---------------
Text is pasted in reverse

Environment data
----------------
Windows 10 Enterprise 10.0.15063

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      5.1.15063.296
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.15063.296
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
![image](https://cloud.githubusercontent.com/assets/5415248/26224277/7290163c-3bd6-11e7-960b-1b3dddfe1572.png)

![image](https://cloud.githubusercontent.com/assets/5415248/26224307/90d2feca-3bd6-11e7-80c0-48e59dc3ba71.png)

I have been able to reproduce this on every machine I've tried on and it appears to exist in PowerShell v6.0.0-alpha.17 Release as well. This doesn't seem to happen with a ctrl+v paste.

**Comment by SteveL-MSFT** [comment]
I didn't get this to repro with beta1, but does repro with Windows PowerShell 5.1.  At this time we're only taking critical security fixes for 5.1.

**Comment by lzybkr** [comment]
This is https://github.com/lzybkr/PSReadLine/issues/496

**Comment by jzabroski** [comment]
Was this ever actually fixed?  I am on PowerShell 7.0.0 and still see this issue.  In my scenario, I have some PowerShell scripts documented in Confluence as part of developer on-boarding.  Confluence saves all documentation with <kbd>LF</kbd> and so when I paste a script from Confluence into PowerShell, it reverses the whole script.

In addition, I tried running `Update-Module PSReadLine` based on @lzybkr  saying this is a PSReadLine issue, and I got the following error:

```powershell
PS D:\Users\John.Zabroski> update-module PSReadLine
Update-Module: Module 'PSReadLine' was not installed by using Install-Module, so it cannot be updated.
```
